src/features/counselorSlice.js

```diff
@@ -0,0 +1,80 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+export const getInfo = createAsyncThunk("counselor/getInfo", async (userId) => {
+  const { data } = await axios.get(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${userId}`,
+    userId,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return data.data.counselor;
+});
```

Comment:
```suggestion
export const getCounselorInfo = createAsyncThunk("counselor/getCounselorInfo", async (userId) => {
  const { data } = await axios.get(
    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${userId}`,
    userId,
    {
      withCredentials: true,
    }
  );

  return data.data.counselor;
});
```
어떤 Info를 가져오는지 명확하게 변수명을 지어 사용하면 좋을 것 같아요!